static/main.css

```diff
@@ -313,12 +344,30 @@ main.normalize {
     unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
 
-@media only screen and (max-width: 735px) {
+/* latin */
```

Comment:
No need for the comment, it's in the file name :)

Reply:
This maintains consistency with the rest of the fonts; see directly above.